components/script/dom/xmlhttprequest.rs

```diff
@@ -1009,14 +1013,14 @@ impl XMLHttpRequest {
                 let upload_complete = &self.upload_complete;
                 if !upload_complete.get() {
                     upload_complete.set(true);
-                    self.dispatch_upload_progress_event(atom!("progress"), None);
+                    self.dispatch_upload_progress_event(atom!("event"), None);
```

Comment:
This [changes](https://github.com/whatwg/xhr/pull/90/files#diff-1feda49b40370635faef8b655f144f64L1259) removes the progress event named ```progress```.